docs/examples/timer_operations.rst

```diff
@@ -15,33 +14,57 @@ define our recurring Timer job.
         NativeAppAuthClient,
         TimerClient,
         TimerJob,
-        TransferClient,
         TransferData,
     )
+    from globus_sdk.scopes import (
+        GCSCollectionScopeBuilder,
+        MutableScope,
+        TimerScopes,
+        TransferScopes,
+    )
 
     GOEP1 = "ddb59aef-6d04-11e5-ba46-22000b92c6ec"
     GOEP2 = "ddb59af0-6d04-11e5-ba46-22000b92c6ec"
+    NATIVE_CLIENT_ID = "61338d24-54d5-408f-a10d-66c06b59f6d2"
     TIMER_CLIENT_ID = "524230d7-ea86-4a52-8312-86065a9e0417"
-    TIMER_SCOPE = f"https://auth.globus.org/scopes/{TIMER_CLIENT_ID}/timer"
-    TRANSFER_SCOPE = "urn:globus:auth:scope:transfer.api.globus.org:all"
-    SCOPES = [TIMER_SCOPE, TRANSFER_SCOPE]
+
+    # If any of the collections you're targeting are mapped collections
+    # that require data_access scopes, include them in this list
+    MAPPED_COLLECTION_IDS = []
+
+    # Build a scope that will give the Timer service
+    # access to perform transfers on your behalf
+    timer_scope = TimerScopes.make_mutable("timer")
+    transfer_scope = TransferScopes.make_mutable("all")
+    transfer_action_provider_scope = MutableScope(
+        "https://auth.globus.org/scopes/actions.globus.org/transfer/transfer"
+    )
+
+    # If you declared and mapped collections above, add them to the transfer scope
+    if MAPPED_COLLECTION_IDS:
+        for id in MAPPED_COLLECTION_IDS:
+            gcs_data_access_scope = GCSCollectionScopeBuilder(id).make_mutable(
+                "data_access",
+                optional=True,
+            )
+            transfer_scope.add_dependency(gcs_data_access_scope)
+        transfer_action_provider_scope.add_dependency(transfer_scope)
```

Comment:
Should we maybe dedent this block and remove the `if MAPPED_COLLECTION_IDS` check?
As it currently reads, I got a little mixed up and thought there was a possibility that this `transfer AP -> transfer` dependency could be missing.

But I think that the Transfer AP depends on Transfer statically? So it would do no harm to add the dependency, but it would maybe reduce the number of distinct cases the user has to think about.

Reply:
I tested this out to confirm, and you're right, there shouldn't be an issue with just allowing it to be added regardless of whether there are mapped collections. Reads better! 👍